packages/space-opera/src/components/materials_panel/materials_panel.ts

```diff
@@ -319,6 +513,23 @@ export class MaterialPanel extends ConnectedLitElement {
         this.onClick(event);
       }
     });
+
+    const self = this;
```

Comment:
Can you  explain what's  going on here?

Reply:
Comments added to the code, update OnHandleValue to OnOK.